src/base/profile.cpp

```diff
@@ -58,9 +55,15 @@ void Profile::initialize(const QString &rootProfilePath, const QString &configur
     m_instance = new Profile(profile.release(), converter.release());
```

Comment:
You need also check for nullptr before create it.

Reply:
oops, fixed.